src/C1T2X_OBU.py

```diff
@@ -185,6 +186,13 @@ def LAN_listening_thread():
 		error = True
 		c1t2x_logger.info("Terminating LAN Thread")
 
+def strip_header(packet):
```

Comment:
I would make sure to add a comment that specifies that this removes unnecessary RSU header information. Also, best to link/reference the original code which is found here: https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/engineering_tools/msgIntersect.py